impeller/renderer/backend/vulkan/render_pass_cache_unittests.cc

```diff
@@ -11,7 +11,41 @@
 namespace impeller {
 namespace testing {
 
-//
+using RendererTest = PlaygroundTest;
```

Comment:
@dnfield  See here. For some reasonable using DISABLE_ caused a problem with the test harness so I just commented it out.